src/main/groovy/life/qbic/datamodel/datasets/OxfordNanoporeMeasurement.groovy

```diff
@@ -368,25 +374,4 @@ final class OxfordNanoporeMeasurement {
     String getRelativePath() {
         return this.measurementFolder.relativePath
     }
-
-    /*
-    Inner class that contains the logic for the metadata validation
-     */
-    private static class MetaData {
-
-        private static final SCHEMA = "/schemas/ont-metadata.schema.json"
-
-        static validateMetadata(Map metaData) throws ValidationException {
-            // Load schema
-            final def metaDataJson = new JSONObject(metaData)
-            final def schemaStream = OxfordNanoporeMeasurement.getResourceAsStream(SCHEMA)
-            final def rawSchema = new JSONObject(new JSONTokener(schemaStream))
-            final def jsonSchema = SchemaLoader.load(rawSchema)
-            // Validate against schema
-            jsonSchema.validate(metaDataJson)
-        }
-
-    }
-
-
```

Comment:
So this means we cannot make use of the schemata we defined?

Reply:
Yes we can but in the core-utils lib where the parsers reside and not in the data-model-lib